internal/journal/voljournal.go

```diff
@@ -470,7 +470,7 @@ func reserveOMapName(ctx context.Context, monitors string, cr *util.Credentials,
 				continue
 			}
 
-			return "", err
+			return "", fmt.Errorf("failed to create object: %w", err)
```

Comment:
you can better place it as `failed to create omap object for `oMapNamePrefix+iterUUID`